packages/wonder-blocks-dropdown/components/action-item.js

```diff
@@ -164,6 +164,7 @@ const styles = StyleSheet.create({
         flexDirection: "row",
         alignItems: "center",
         display: "flex",
+        margin: 0,
```

Comment:
This would actually be fixed after the PR to reset `button` styles in Safari gets merged.

Reply:
@wishdasher I've merged #222.